### PR TITLE
Add SCSS media mixins #326

### PR DESCRIPTION
**Pull request by Kierancz**
Add SCSS mixins for bootstrap breakpoint sizes

https://github.com/codefordenver/Circular/issues/326

### CODE REVIEW

**Comment by hoop71** [comment]
@Kierancz Let me know if you need help clearing the failed tests

**Comment by Kierancz** [comment]
@hoop71 thanks, it seems the issue is my Google maps API key I got from Katie doesn't want to work. Keeps throwing an InvalidKeyMapError. Could you send me the key you're using? My email is kierwinski@gmail.com or you can find me on flowdock. Thanks!

**Comment by Kierancz** [comment]
Huh it's weird because all the tests pass in dev and there's nothing to commit. Don't see any errors in the console either, except one when the map is opened about maps being loaded more than once. The code I added shouldn't have failed any tests that weren't already failing anyway. Is there a way I can run travis locally? Any suggestions?

**Comment by hoop71** [comment]
Check the Travis error here. I haven't looked at your code yet but can if this doesn't help. Looks to be something with the 'Learn More First" button.

https://travis-ci.org/codefordenver/Circular/builds/402069857

File: client/src/components/CollapsePanel.js [hunk 1]
```diff
@@ -19,19 +19,8 @@ class CollapsePanel extends Component {
   render() {
     const { body, titleText } = this.props;
     return (
-      /* eslint-disable */
-      <div onClick={this.togglePanelExpanded}>
-        {/* eslint-enable */}
-        <Panel
-          /*
-          bsStyle="remove-default" causes an console error but it
```
Comment (hoop71) [inline client/src/components/CollapsePanel.js]: YESSSS! @TheBrooks

**Comment by hoop71** [comment]
@Kierancz  Please squash and merge when you get a chance :)

**Pull request merged**